src/Sylius/Bundle/TaxationBundle/Resources/translations/validators.pl.yml

```diff
@@ -16,6 +16,7 @@ sylius:
             not_blank: Proszę podać nazwę stawki podatkowej.
         amount:
             not_blank: Proszę podać wartość stawki podatkowej.
+            invalid: Wartość stawki podatkowej jest nieprawidłowa.
```

Comment:
We should not change any non-English translations in the PR and let Crowdin to its job 🚀